spec/item_repository_spec.rb

```diff
@@ -144,5 +144,95 @@
 
       expect(ir.find_all_by_price_in_range('-3'..'-1')).to eq([])
     end
+    end
+    describe '#find_all_by_price_in_range' do
+      it 'returns an array of all items with a price found in the range' do
+        se = SalesEngine.from_csv(
+          :items     => './data/items.csv',
+          :merchants => './data/merchants.csv'
+        )
+
+        ir = se.items
+
+        expect(ir.find_all_by_price_in_range('100'..'1000')[5].name).to eq('Two tone blue stoneware pot')
+      end
+      it 'returns an empty array when no items are found' do
```

Comment:
Starting with the 'end' on line 147 through the rest of the file, it looks like everything is indented by 2 extra spaces.